src/puppetlabs/trapperkeeper/authorization/ring.clj

```diff
@@ -2,6 +2,10 @@
   (:require [schema.core :as schema])
   (:import (java.security.cert X509Certificate)))
 
+(def cert-key
+  "The nested key where the client certificate for the request is stored"
+  [:authorization :certificate])
+
 (def is-authentic-key
   "The nested key where authenticity information is stored."
   [:authorization :authentic?])
```

Comment:
I don't see hardly any value in these `def`s since we're not consistently using this pattern; we just dig into the map directly in most places.